### ISSUE TITLE
False positive for TC003

### ISSUE THREAD

**Issue by kurtschelfthout** [snok/flake8-type-checking#102]
Second and last one:

```python
import boto3.s3.transfer

if __name__ == "__main__":
    print(f"hello there: {boto3.s3.transfer.TransferConfig()}")
```

Triggers TC003 on the import.  Changing it to from ... import works.
